src/v2/Apps/Order/Routes/Shipping/index.tsx

```diff
@@ -192,8 +194,8 @@ export class ShippingRoute extends Component<ShippingProps, ShippingState> {
 
     return (
       this.state.shippingOption === "SHIP" &&
-      processWithArtaShipping &&
-      shippingCountry === "US"
+      ((processWithArtaShipping && shippingCountry === "US") ||
+        (artsyShippingInternational && shippingCountry !== "US"))
```

Comment:
Just for my own learning 🧠 :

Am I correct in saying will be relying on validations on the CMS side of things to handle checks of where the artwork is located? Assuming the protection to only allow US exports live there?